### DISCUSSION TITLE
Timeline

### DISCUSSION THREAD

**Discussion by spupuz** [roflcoopter/viseron#772] [Ideas]
Hello, in my view the timeline view should be a view about the list of all events/recordings coming from all cameras, this fit correctly where the option of recording when motion is detected, what do you think?

**Comment by spupuz** [comment]
@roflcoopter what do you think?

**Reply by ItsMeRitch** [reply]
This is already kind of the case whereby if you have a motion detector enabled and also 24/7 recordings, the timeline will allow you to scrub through all footage but also show events like movements. 

I get what you mean about having a single timeline and then showing all cameras however, in my case, my timeline would be so cluttered as some of my cameras overlap too. :)

**Comment by roflcoopter** [comment]
The Events tab could for sure allow you to select multiple cameras, i do think it will be very hard to make the same thing for the timeline.

The tricky part would be to synchronize all the streams since they can have gaps in them at any point

**Comment by roflcoopter** [comment]
In 3.0.0b10 it is possible to select multiple cameras and have them show in a grid.

Will work on syncing the streams for the next release
